render-test/runner.cpp

```diff
@@ -562,12 +572,35 @@ bool TestRunner::runOperations(const std::string& key, TestMetadata& metadata) {
             const mbgl::JSValue* propertyValue = &operationArray[3];
             layer->setLayoutProperty(propertyName, propertyValue);
         }
-    // probeMemoryStart
+    } else if (operationArray[0].GetString() == fileSizeProbeOp) {
+        // probeFileSize
+        assert(operationArray.Size() >= 3u);
+        assert(operationArray[1].IsString());
+        assert(operationArray[2].IsString());
+
+        std::string mark = std::string(operationArray[1].GetString(), operationArray[1].GetStringLength());
+        std::string path = std::string(operationArray[2].GetString(), operationArray[2].GetStringLength());
+        assert(!path.empty());
+
+        uintmax_t size = 0;
+        std::string absolutePath = path[0] == '/' ? path : metadata.paths.defaultExpectations() + path;
```

Comment:
use `mbgl::filesystem::path::is_absolute()`?
(could add a method returning `mbgl::filesystem::path` instead of `std::string defaultExpectations()` )